src/app/ui-helper/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -49,8 +50,22 @@ export class EntitySubrecordComponent implements OnInit, OnChanges {
     }
   }
 
-
+  /**
+   * 
+   * @param record uses the input from the form field to create a record
+   * curently has ugly workaround for entity type user because the id is dependent from the users input
+   * this workaround is highly WIP because its buggy aswell for creating a user twice somehow. 
+   */
   save(record: Entity) {
+    if(record.getType()==='User'){
+      let user: User;
+      user=new User((record as User).name);
```

Comment:
Don't you lose the "_rev" field of an existing user object when creating a new user like this? That would make PouchDB reject changing it.